src/pages/components/style.scss

```diff
@@ -1,4 +1,5 @@
 @import '@/assets/scss/style.scss';
+@import '../../assets/scss/buttons.scss';
```

Comment:
Please use import with @ to be consistent

Reply:
This import already have @, I don't know what you mean

Reply:
Done